### ISSUE TITLE
ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by Prefinem** [charliecochran/async-react-hooks#2]
```ReferenceError: regeneratorRuntime is not defined
    at <project_dir>/node_modules/async-react-hooks/dist/index.js:116:3
    at Object.<anonymous> (<project_dir>/node_modules/async-react-hooks/dist/index.js:169:2)
    at Module._compile (internal/modules/cjs/loader.js:868:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:879:10)
    at Module.load (internal/modules/cjs/loader.js:731:32)
    at Function.Module._load (internal/modules/cjs/loader.js:644:12)
    at Module.require (internal/modules/cjs/loader.js:771:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.async-react-hooks (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:6294:18)
    at __webpack_require__ (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:23:31)
    at Module../components/inactive-device-emailer/EmailButtons.js (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:177:75)
    at __webpack_require__ (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:23:31)
    at Module../pages/inactive-device-emailer/index.js (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:6096:106)
    at __webpack_require__ (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:23:31)
    at Object.4 (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:6271:18)
    at __webpack_require__ (<project_dir>/.next/server/static/development/pages/inactive-device-emailer.js:23:31)```

**Comment by charliecochran** [comment]
Should be fixed in v1.1. I updated to use just Promises instead of async/await rather than force a dependency on babel runtime.

**Comment by Prefinem** [comment]
Awesome!  I will give it another go
